ruby_on_rails/app_initialisation.md

```diff
@@ -63,17 +67,6 @@ They are always idempotent (runnable multiple times).
   chmod +x bin/run bin/fastcheck bin/check
   ```
 
-* `bin/setup` was generated by Rails. We'll add a `pre-push` hook to it:
-
-  ```ruby
-  unless File.exist?('.git/hooks/pre-push')
-    system 'ln -s ../../bin/fastcheck .git/hooks/pre-push'
-  end
-  ```
-
-  Also change `system!('bundle install')` to `system!('bundle install --jobs=3 --retry=3')`
```

Comment:
no need to specify the jobs, bundler already defines it based on the amount of processors. retry is also not really necessary, we can live without it and simplify our setup guide